config/src/projects/nahmii.ts

```diff
@@ -0,0 +1,95 @@
+import { Project } from './Project'
+
+export const nahmii: Project = {
+  name: 'Nahmii 1.0',
+  slug: 'nahmii',
+  bridges: [
+    {
+      address: '0xcc8d82f6ba952966e63001c7b320eef2ae729099',
+      sinceBlock: 6921581,
+      tokens: ['NII', 'WETH', 'USDC', 'USDT'],
```

Comment:
```suggestion
      tokens: ['NII'],
```
I run the code and the other tokens never touched the contract